### DISCUSSION TITLE
本科毕设参考文献引用问题

### DISCUSSION THREAD

**Discussion by Zhuzy0608** [note286/xduts#117] [Q&A] [answered]
xdusetup里面添加了这两句
`style/bib-backend = bibtex,
info/bib-resource={refers.bib}`

正文中引用部分编译后是问号，并且参考文献那一页也没有出现bib文件里面的参考文献。
`随着集成电路技术的发展\cite{WXYJ201607004}`

```
@article{WXYJ201607004,
 author = {侯斌 & 莫亭亭},
 title = {用于16 bit SAR ADC的高精度比较器的设计},
 journal = {微电子学与计算机},
 volume = {33},
 number = {15-18+23},
 year = {2016},
 issn = {1000-7180},
 doi ={10.19304/j.cnki.issn1000-7180.2016.07.004}
 }
```

**Comment by Zhuzy0608** [comment]
![image](https://user-images.githubusercontent.com/130884066/232276157-cb85d31c-86cd-465d-85fd-b11bcf8f89a1.png)

**Comment by Zhuzy0608** [comment]
编译方式我选择xe-bib-xe-xe

**Comment by note286** [comment, marked as answer]
`author = {侯斌 and 莫亭亭},`。
